packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -772,32 +772,116 @@ export class Stack extends Construct implements ITaggable {
    *
    * @internal
    */
-  public _addAssemblyDependency(target: Stack, reason?: string) {
```

Comment:
this is a breaking change to a public method (`reason: string` -> `reason: StackDependencyReason`)

Reply:
This is `@internal`, so maybe this is fine?